authenticate/authenticate.cabal

```diff
@@ -8,7 +8,7 @@ synopsis:        Authentication methods for Haskell web applications.
 description:     API docs and the README are available at <http://www.stackage.org/package/authenticate>.
 category:        Web
 stability:       Stable
-cabal-version:   >= 1.6
+cabal-version:   >= 1.8
```

Comment:
Can you add a minor version bump and changelog entry and changelog entry for this?

Reply:
Done